### PR TITLE
Update face forward extension

### PR DESCRIPTION
**Pull request by TheGemDev**
The face forward extension is excellent but sometimes fails to get the default direction, adding a default direction property fixes the issue. 

image

### CODE REVIEW

**Comment by TheGemDev** [comment]
I should have named it named it default angle.

**Comment by Bouh** [comment]
> I should have named it named it default angle.

- Is "Offset angle" is more appropriate?

- Please remove the dot at the end of the short description.

File: extensions/reviewed/FaceForward.json [hunk 1]
```diff
@@ -421,15 +384,27 @@
...
           "description": "",
+          "group": "",
           "extraInformation": [],
           "hidden": false,
           "name": "RotationSpeed"
         },
+        {
+          "value": "0",
+          "type": "Number",
+          "label": "Default direction.",
```
Comment (Bouh) [inline extensions/reviewed/FaceForward.json:395]: Remove the dot here.

File: extensions/reviewed/FaceForward.json [hunk 2]
```diff
@@ -421,15 +384,27 @@
           "type": "Number",
           "label": "Rotation speed (degrees per second).  Use \"0\" for immediate turning.",
```
Comment (Bouh) [inline extensions/reviewed/FaceForward.json:385]:
```suggestion
          "label": "Rotation speed (degrees per second).  Use \"0\" for immediate turning",
```

File: extensions/reviewed/FaceForward.json (hunk 1 above)
Comment (Bouh) [inline extensions/reviewed/FaceForward.json:395]:
```suggestion
          "label": "Default direction",
```

**Comment by TheGemDev** [comment]
fixed

**Comment by tristanbob** [comment]
@TheGemDev great idea to add this feature!  I'm merging this now so people can start benefiting from it. :)

**Pull request merged**